src/index.js

```diff
@@ -0,0 +1,168 @@
+import { differenceCiede2000 } from 'd3-color-difference';
+
+export default class toColor {
+  HUE_MAX = 360;
+
+  constructor(seed, options) {
+    this.options = options || {};
+    if (typeof seed === 'string' || typeof seed === 'number') {
+      this.seed = typeof seed === 'string' ? this._stringToInteger(seed) : seed;
+    } else {
+      throw new TypeError('Seed value must be a number or string');
+    }
+
+    this.known = [];
+  }
+
+  getColor(count = 0) {
+    const h = this._pickHue();
+    const s = this._pickSaturation(h);
+    const b = this._pickBrightness(h, s);
+    const hsl = this._HSVtoHSL(h, s, b);
+    const formatted = this._formatHSL(hsl);
+    const PASSABLE_DISTANCE = 50; 
+
+    // The larger `count` grows, we need to divide actual distance to avoid
+    // hitting a maxiumum call stack error.
+    const ACTUAL_DISTANCE = PASSABLE_DISTANCE / Math.pow(1.25, count);
```

Comment:
The distance scaling seems _a little_ harsh - by 10 or 11th iteration we're at pretty hard to distinguish differences.

Reply:
I can lower `1.25` to slow division more but this is a pretty sensitive dial